spec/factories/new_book_suggestion.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :book_suggestion do
+    user { create(:user) }
+    author Faker::Book.author
```

Comment:
Surround every attribute value with curly braces `{}` to get a different value on each factory call